app/views/layouts/application.html.erb

```diff
@@ -70,6 +70,28 @@
               </li>
             </ul>
           <% end %>
+
+          <% if current_namespace == 'provider_interface' %>
+          <ul id="navigation" class="govuk-header__navigation " aria-label="Top Level Navigation">
+            <li class="govuk-header__navigation-item">
+              <%= link_to "Sign out (#{Faker::Name.first_name} #{Faker::Name.last_name})", '#', class: 'govuk-header__link' %>
+            </li>
+          </ul>
+          <% end %>
+
+          <% if current_namespace == 'support_interface' %>
+          <ul id="navigation" class="govuk-header__navigation " aria-label="Top Level Navigation">
+            <li class="govuk-header__navigation-item">
+              <%= link_to 'Applications', support_interface_path, class: 'govuk-header__link' %>
+            </li>
+            <li class="govuk-header__navigation-item">
+              <%= link_to 'API tokens', support_interface_tokens_path, class: 'govuk-header__link' %>
+            </li>
+            <li class="govuk-header__navigation-item">
+              <%= link_to "Sign out (#{Faker::Name.first_name} #{Faker::Name.last_name})", '#', class: 'govuk-header__link' %>
+            </li>
+          </ul>
+          <% end %>
         </div>
```

Comment:
possible future improvement: put different namespaces in different partials ('navigation_for_candidate`, `navigation_for_provider` ...)

Reply:
I agree this is a bit gnarly. Could [Nested Layouts](https://guides.rubyonrails.org/layouts_and_rendering.html#using-nested-layouts) help? (We add a `yield` in `application.html.erb` and provide `content_for` in e.g. `layouts/candidate_interface.html.erb`)

Reply:
I think the layouts are probably close enough so that we can use partials like @alexdesi says. Let's do that once we actually look at the header & footer though, I'm not sure they've been fully designed.